frontend/packages/core/src/stories/radio-group.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import type { Meta } from "@storybook/react";
+
+import type { RadioGroupProps } from "../Input/radio-group";
+import { RadioGroup } from "../Input/radio-group";
+
+export default {
+  title: "Core/Input/RadioGroup",
+  component: RadioGroup,
+  argTypes: {
+    onChange: { action: "onChange event" },
+  },
+} as Meta;
+
+const Template = (props: RadioGroupProps) => <RadioGroup {...props} />;
+
+export const Primary = Template.bind({});
+Primary.args = {
+  name: "continue",
+  label: "Favorite color",
+  options: [{ label: "red" }, { label: "green" }, { label: "blue" }],
+};
```

Comment:
It might be good to have a story that demonstrates options that have unique values

Reply:
added